distribution/scripts/ballerina/file_connector/main.bal

```diff
@@ -0,0 +1,26 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/file;
+
+configurable string epKeyPath = ?;
+configurable string epKeyPassword = ?;
+
+const FILE_NAME = "sample.json";
+
+listener http:Listener securedEP = new (9090,
+    secureSocket = {
+        key: {
+            path: epKeyPath,
+            password: epKeyPassword
+        }
+    }
+);
+
+service /fileConnector on securedEP {
+    resource function post .(@http:Payload json payload) returns json|error {
+        check io:fileWriteJson(FILE_NAME, payload);
```

Comment:
When we are running this in a docker environment, we need to have access to this directory to read, write and remove right? How do we configure the cloud options for these services?

Reply:
Assuming we are running in a vm and we don't have permission issues as per offline discussion we can keep it as it. Decided to make this a configurable